src/devices_models/devices/common/constraints_structs.jl

```diff
@@ -1,15 +1,12 @@
 """ Data Container to construct range constraints"""
 struct DeviceRange
-    names::Vector{String}
-    values::Vector{MinMax}
-    additional_terms_ub::Vector{Vector{Symbol}}
-    additional_terms_lb::Vector{Vector{Symbol}}
+    limits::MinMax
+    additional_terms_ub::Vector{Symbol}
+    additional_terms_lb::Vector{Symbol}
 end
 
-function DeviceRange(count::Int64)
-    names = Vector{String}(undef, count)
-    limit_values = Vector{MinMax}(undef, count)
-    additional_terms_ub = fill(Vector{Symbol}(), count)
-    additional_terms_lb = fill(Vector{Symbol}(), count)
-    return DeviceRange(names, limit_values, additional_terms_ub, additional_terms_lb)
+struct DeviceTimeSeries
```

Comment:
If you keep it as a struct per device, you only need to pass a single struct, I suggest making a single struct as the input. I am not clear why the bus number is needed. 

```suggestion
struct DeviceTimeSeries
name::String
limits::MinMax
additional_terms_ub::Vector{Symbol}
additional_terms_lb::Vector{Symbol}
multiplier::Float64
timeseries::Vector{Float64}
end
```
Even if there is a little repetition, it is far cleaner to define a single struct to the constraints that contains the required information.